### PR TITLE
INSE-540174: Paris Schema Validation

### PR DESCRIPTION
**Pull request by Thomas-Boyle**
Added schema files and environment variable for Paris validation.

### CODE REVIEW

File: shared_libs/hl7_validation/tests/test_paris.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import unittest
+
+from hl7_validation import validate_er7_with_flow
+from hl7_validation.validate import XmlValidationError
+
+
+class TestParisValidation(unittest.TestCase):
+    def test_paris_a05_schema_validation_working(self) -> None:
```
Comment (iichr) [inline shared_libs/hl7_validation/tests/test_paris.py:8]: [nit] schema validation should always be working - it can be successful or fail, would rename these two scenarios to validation_successful

Comment (iichr) [reply shared_libs/hl7_validation/tests/test_paris.py:8]: also in the naming, here you use the A05 message structure but in the failure scenarios it's the trigger event A28 - which one's better for consistency?

File: shared_libs/hl7_validation/tests/test_paris.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            "EVN||20250619115001||||20250619115001",
+            "XXX|This is an invalid segment not in the schema",
+            "PID|||700001^^^169^PI||TEST^Patient^^^^^U||20000101|M|||1 TEST STREET^^^TEST^CF11 9AD||~900001|||||||||Z",
+            "PV1||N"
+        ])
+
+        with self.assertRaises(XmlValidationError):
+            validate_er7_with_flow(er7, "paris")
+
+    def test_paris_a28_correctly_structured_invalid_data_validation_failure(self) -> None:
```
Comment (iichr) [inline shared_libs/hl7_validation/tests/test_paris.py:40]: what is invalid in here, could we clarify in the test naming - missing something mandatory, invalid date ...? 
for the previous test case it's very obvious

**Review by iichr — COMMENTED** [review]
Not had a chance to look at the full implementation that went in, are the validation error messages explicit about which fields caused a failure? if so, would be good to assert in the existing unit tests with the meaningful error messages.

Any more edge cases for Paris that are worth unit testing? PIMS and Chemocare come to mind with their `""` values in some fields of string/date type!

File: shared_libs/hl7_validation/tests/test_paris.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        er7 = "\r".join([
+            "MSH|^~\\&|169|169|100|100|20250619115001||ADT^A28^ADT_A05|185352620250619115001|P|2.5.1|||||GBR||EN",
+            "EVN||20250619115001||||20250619115001",
+            "XXX|This is an invalid segment not in the schema",
+            "PID|||700001^^^169^PI||TEST^Patient^^^^^U||20000101|M|||1 TEST STREET^^^TEST^CF11 9AD||~900001|||||||||Z",
+            "PV1||N"
+        ])
+
+        with self.assertRaises(XmlValidationError):
+            validate_er7_with_flow(er7, "paris")
```
[the remark this answers is not in the document]
Comment (iichr) [reply shared_libs/hl7_validation/tests/test_paris.py:38]: We want to assert that there is indeed an error _with the appropriate reason_ being passed to the caller (hl7server) from the place where that error originates (here) to aid us/the support team in debugging should an invalid message be provided

**Review by iichr — APPROVED** [review]
